packages/postgres-database/src/simcore_postgres_database/utils_tags_sql.py

```diff
@@ -183,6 +203,11 @@ def add_tag_to_project_stmt(*, project_index: int, tag_id: int):
     )
 
 
+#
+# SERVICE TAGS
+#
+
+
 def get_tags_for_services_stmt(*, key: str, version: str):
     return sa.select(services_tags.c.tag_id).where(
```

Comment:
Note: If tags are or will be part of the "ServiceGet" response for the frontend, I would suggest including them directly in the service listing query for performance reasons. This way, we avoid making separate database calls for each service. (While working with the Project resource, I've noticed that we often do this, and what should ideally be a single database call can easily escalate to, for example, 250+ calls—I can show you in person.)